styles.css

```diff
@@ -147,5 +444,152 @@ section {
     background-position: 100% 100%;
   }
 
- 
+  .container h1 {
+    font-family: 'Poppins' "Trebuchet MS", "Lucida Sans Unicode", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 48px;
+    line-height: 60px;
+    display: flex;
+    align-items: center;
+    letter-spacing: 0.37px;
+    color: #091e42;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+  }
+
+  .work_content {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+  }
```

Comment:
- [ ] Your cards look great so far :smile: However for your design to look exactly like the Figma design, you will have to increase the size of your cards a little :+1: This is the size according to the Figma design 
![image](https://user-images.githubusercontent.com/102699958/230460396-ddf7d1bd-4c26-4930-85ec-298c3ffcdf36.png)
- [ ] So kindly apply this to your design and ensure that the items in each card are properly arranged and spaced like that of the Figma design :+1: 
- [ ] Also the list items `CANOPY, Back End Dev, 2015` aren't properly styled. Kindly refer to the design guidelines and ensure that your design looks similar to the Figma design 

| Your Design | Figma Design |
|--------|--------|
| ![image](https://user-images.githubusercontent.com/102699958/230460740-557f3aea-bffc-45e4-a45c-f47169be44f1.png) | ![image](https://user-images.githubusercontent.com/102699958/230460853-cdbaabf3-a649-4703-935c-255907316747.png) |